src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -37,6 +37,14 @@ The input options below have been removed in v6 from Storage APIs:
 
 >Note: We have remapped all capitalized input and output keys to "camelCase" in v6.
 
+## Deprecated Options
+
+Starting version 6.x.x, the following parameters in input and output options have been marked deprecated and will be removed in a future version.
+
+- `key`: _A new `path` parameter is introduced and will be used instead of `key` parameter in input and output options. The `path` parameter allows for flexible input string or path construction based on user's identity id._
```

Comment:
```suggestion
- `key`: _A new `path` parameter has been introduced and will be used instead of the `key` parameter in input and output options. The `path` parameter allows for flexible path construction based on user's identity id._
```